src/WebJobs.Script/Workers/Rpc/IJobHostRpcWorkerChannelManager.cs

```diff
@@ -9,12 +9,14 @@ namespace Microsoft.Azure.WebJobs.Script.Workers.Rpc
 {
     internal interface IJobHostRpcWorkerChannelManager
     {
-        void AddChannel(IRpcWorkerChannel channel);
+        void AddChannel(IRpcWorkerChannel channel, string language);
 
         Task<bool> ShutdownChannelIfExistsAsync(string channelId, Exception workerException);
 
         void ShutdownChannels();
 
+        IEnumerable<IRpcWorkerChannel> GetChannels(string language);
+
         IEnumerable<IRpcWorkerChannel> GetChannels();
```

Comment:
Can we get rid of `GetChannels()`?

Reply:
I think `GetChannels()` is returning _all channels_ of all languages, so if we can't get rid of it then probably renaming it to `GetAllChannels()` and the other one to `GetLanguageChannel()`

Reply:
`GetChannels` previously used to return all channels, i added a overloaded function with language to avoid changes in places where `GetChannels` is used. It's still used in some cases.